services/registration/controller/controller.go

```diff
@@ -351,13 +351,13 @@ func UpdateCurrentMentorRegistration(w http.ResponseWriter, r *http.Request) {
 	err = service.UpdateMentorRegistration(id, mentor_registration)
 
 	if err != nil {
-		panic(errors.UnprocessableError(err.Error()))
+		panic(errors.InternalError(err.Error(), "Could not update mentor registration.\n"+err.Error()))
```

Comment:
Don't need a newline here and don't need to append `err.Error()` since it will be included in the `RawError` field.